_docs/javascript-api.md

```diff
@@ -911,6 +911,32 @@ Interceptor.attach(Module.findExportByName("libc.so", "read"), {
 
     -   `depth`: call depth of relative to other invocations
 
+    For eaxmple:
+
+{% highlight js %}
+Interceptor.attach(Module.findExportByName(null, 'read'), {
+    onEnter: function (args) {
+        console.log('Context information: ')
```

Comment:
The latter. And that space inside can be dropped as it will make the line have trailing whitespace (nit).